phpBB/memberlist.php

```diff
@@ -1812,6 +1812,18 @@
 					$memberrow = array_merge($memberrow, $cp_row['row']);
 				}
 
+				/**
+				 * Modify the memberrow data before template variables are assigned.
+				 *
+				 * @event core.memberlist_memberrow_template_vars_before
+				 * @var integer	user_id		The current user ID.
+				 * @var array	row			Array of raw user data.
+				 * @var array	memberrow	Array of member template variables.
+				 * @since 3.3.16-RC1
+				 */
+				$vars = array('user_id', 'row', 'memberrow');
```

Comment:
For the new code, short array notation is recommended:
```suggestion
				$vars = ['user_id', 'row', 'memberrow'];
```